teslajsonpy/homeassistant/charger.py

```diff
@@ -360,7 +360,7 @@ def refresh(self) -> None:
             self.__charge_limit_soc = data["charge_limit_soc"]
             self.__charger_power = data["charger_power"]
             self.attrs["charge_limit_soc"] = self.charge_limit_soc
-            self.attrs["last_reset"] = str(self.last_reset)
+            self.attrs["last_reset"] = self.last_reset
```

Comment:
https://developers.home-assistant.io/docs/core/entity/sensor?_highlight=last_reset#properties

It looks like this is supposed to be a `datetime.datetime`